pkg/component/etcd/poddisruptionbudget/values.go

```diff
@@ -17,11 +17,9 @@ package poddisruptionbudget
 import "k8s.io/apimachinery/pkg/types"
 
 const (
-	ownedByAnnotationKey   = "gardener.cloud/owned-by"
-	ownerTypeAnnotationKey = "gardener.cloud/owner-type"
-	instanceKey            = "instance"
-	nameKey                = "name"
-	appKey                 = "app"
+	instanceKey = "instance"
```

Comment:
These changes are completely unrelated to etcdcopybackuptask and should not be clubbed with this PR. Please remove this and file a separate PR for this.